### PR TITLE
OCM-6448 | fix: Fix actual UX issues

### PR DESCRIPTION
**Pull request by hunterkepley**
- https://issues.redhat.com/browse/OCM-7897
- https://issues.redhat.com/browse/OCM-7896
- https://issues.redhat.com/browse/OCM-7884

Fixes these issues

### CODE REVIEW

File: cmd/rosa-support/create/subnets/cmd.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ var Cmd = &cobra.Command{
 	Short: "Create subnets",
 	Long:  "Create subnets.",
 	Example: `  # Create a pair of subnets with prefix 'mysubnet-' in region 'us-east-2'
-  rosa-support create subnets --name-prefix=mysubnet --region us-east-2 --vpc-id <vpc id> --availability-zones <AZs>`,
```
Comment (chenz4027) [inline cmd/rosa-support/create/subnets/cmd.go:25]: Just wondering, so there's no name-prefix param anymore?

Comment (hunterkepley) [reply cmd/rosa-support/create/subnets/cmd.go:25]: @chenz4027 this functionality was removed from the AWS client (which QE migrated from their own repo), now it is created with an automatically generated prefix within the client IIRC

Comment (chenz4027) [reply cmd/rosa-support/create/subnets/cmd.go:25]: Got it!
/lgtm

**Review by lucasponce — APPROVED** [review]

**Review by gdbranco — APPROVED** [review]

**Pull request merged**